src/Npgsql/TypeHandlers/InternalTypesHandlers/TidHandler.cs

```diff
@@ -21,13 +21,11 @@
 // TO PROVIDE MAINTENANCE, SUPPORT, UPDATES, ENHANCEMENTS, OR MODIFICATIONS.
 #endregion
 
-using System;
-using System.Diagnostics;
-using JetBrains.Annotations;
 using Npgsql.BackendMessages;
 using Npgsql.TypeHandling;
 using Npgsql.TypeMapping;
 using NpgsqlTypes;
+using System.Diagnostics;
```

Comment:
I actually prefer system usings to come first, followed by external packages, followed by my own...